projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/update-flag/update-flag.component.spec.ts

```diff
@@ -438,19 +496,26 @@ describe('UpdateFlagComponent', () => {
   });
 
   it('should display correct title based on the display mode', () => {
-    component.displayContextParameter = '#ARGUMENT(UPDATE)';
-    component.setUpdateCaseFlagTitle(activeFlag);
+    component.displayContextParameter = CaseFlagDisplayContextParameter.UPDATE;
+    // component.setUpdateCaseFlagTitle(activeFlag);
```

Comment:
@johnbenjamin-hmcts - the five commented-out lines can be removed if no longer needed

Reply:
Thank you @Daniel-Lam - I somehow missed removing the comments. I have updated the code now.